addons/point_of_sale/static/src/app/utils/contextual_utils_service.js

```diff
@@ -7,7 +7,6 @@ import {
 } from "@web/core/utils/numbers";
 import { escapeRegExp } from "@web/core/utils/strings";
 import { registry } from "@web/core/registry";
-import { parseFloat } from "@web/views/fields/parsers";
```

Comment:
Can you add a comment in the `formatStrCurrency` that it accepts valueStr that have decimal point, and not decimal based on localization.